lib/engine/game/g_18_al.rb

```diff
@@ -34,6 +34,22 @@ def operating_round(round_num)
           [Step::BuyCompany, blocks: true],
         ], round_num: round_num)
       end
+
+      def revenue_for(route)
+        ensure_terminus_first_or_last(route, %w[A4 Q2])
```

Comment:
this doesn't need to be a new method, just put it all in line

Reply:
I can refactor this in the revenue commit.